src/js/stores/CosmosPackagesStore.js

```diff
@@ -78,7 +82,10 @@ class CosmosPackagesStore extends GetSetBaseStore {
         repositoryAddSuccess: COSMOS_REPOSITORY_ADD_SUCCESS,
         repositoryAddError: COSMOS_REPOSITORY_ADD_ERROR,
         repositoryDeleteSuccess: COSMOS_REPOSITORY_DELETE_SUCCESS,
-        repositoryDeleteError: COSMOS_REPOSITORY_DELETE_ERROR
+        repositoryDeleteError: COSMOS_REPOSITORY_DELETE_ERROR,
+
+        cosmosServiceDescriptionSuccess: COSMOS_SERVICE_DESCRIBE_CHANGE,
```

Comment:
I will rename this. With this name, the StoreMixin names the callback `onCosmosPackagesStoreCosmosServiceDescriptionSuccess()`...which is incredibly wordy.